### PR TITLE
c7n_azure gets a few bits so it can be loaded

### PR DESCRIPTION
**Pull request by stefangordon**

### CODE REVIEW

File: tools/c7n_azure/c7n_azure/entry.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_azure.resources import vm
+from c7n_azure.resources import vnet
+
```
Comment (aluong) [inline tools/c7n_azure/c7n_azure/entry.py]: (Minor) extra new line

Comment (stefangordon) [reply tools/c7n_azure/c7n_azure/entry.py]: I was trying to figure out what the convention was in the project but there were inconsistencies so we should default to https://www.python.org/dev/peps/pep-0008/#blank-lines (two blank lines) - meaning I think in this case this file is correct and query.py is incorrect

Comment (stefangordon) [reply tools/c7n_azure/c7n_azure/entry.py]: Fixed query.py

Comment (kapilt) [reply tools/c7n_azure/c7n_azure/entry.py]: yeah generally we're just holding to running flake8 on files for syntax. pylint is a bit more proscriptive, but not sure we're ready there.

**Pull request merged**